packages/discord.js/src/client/actions/MessageDeleteBulk.js

```diff
@@ -33,8 +33,9 @@ class MessageDeleteBulkAction extends Action {
        * Emitted whenever messages are deleted in bulk.
        * @event Client#messageDeleteBulk
        * @param {Collection<Snowflake, Message>} messages The deleted messages, mapped by their id
+       * @param {Channel} channel The channel that the messages were deleted in
```

Comment:
This can be more accurate.

```suggestion
       * @param {TextBasedChannels} channel The channel that the messages were deleted in
```